cts/aggregation/stages/out.json

```diff
@@ -0,0 +1,23 @@
+{
+  "simple": {
+    "request": {
+      "aggregate": "values",
+      "pipeline": [
+        {
+          "$out": "output_collection"
+        }
+      ],
+      "$db": "{{.Database}}"
+    },
+    "response": {
+      "ok": {
+        "$numberDouble": "0.0"
+      },
+      "errmsg": "The 'cursor' option is required, except for aggregate with the explain argument",
```

Comment:
I think you want to produce `$out` response, not error

Reply:
My bad, fixed